pauthabielf64/pauthabielf64.rst

```diff
@@ -853,34 +853,36 @@ The (platform id, version number) of (0, 0) is reserved as an invalid
 combination. The program cannot be run when pointer authentication is
 enabled.
 
-The version id in ``.note.AARCH64-PAUTH-ABI-tag`` is not directly
-related to the version number of this document. It is controlled by
-the object-producer based on the signing schema that have been used
-for pointers.
+The version is not directly related to the version number of this
+document. It is controlled by the object-producer based on the signing
+schema that has been used for pointers.
 
 Base Compatibility Model
 ------------------------
 
 A per-ELF file marking scheme permits a coarse way of reasoning about
 compatibility.
 
-* The absence of a ``.note.AARCH64-PAUTH-ABI-tag`` section means no
-  information on how pointers are signed is available for this ELF
-  file.
+* The absence of ``GNU_PROPERTY_AARCH64_FEATURE_PAUTH``
+  means no information on how pointers are signed is available for this
+  ELF file.
 
-* The presence of a ``.note.AARCH64-PAUTH-ABI-tag`` means that the
+* The presence of ``GNU_PROPERTY_AARCH64_FEATURE_PAUTH``
+  with (platform id, version number) other than (0, 0) means that the
   pointers were signed in a compatible way with the default signing
   rules for tuple (platform id, version number).
 
-* The static linker may fault the combination of relocatable
-  objects that contain ``.note.AARCH64-PAUTH-ABI-tag`` sections with
-  incompatible (platform id, version number) tuples. If an ELF file is
+* The static linker may fault the combination of relocatable objects
+  that contain ``GNU_PROPERTY_AARCH64_FEATURE_PAUTH``
+  with incompatible (platform id, version number) tuples. If an ELF file is
   produced, the output ``.note.AARCH64-PAUTH-ABI-tag`` must have the
```

Comment:
Here we also should say ``GNU_PROPERTY_AARCH64_FEATURE_PAUTH``.

Reply:
Ah, yes, thanks!